src/main/java/com/gamingmesh/jobs/config/ConfigManager.java

```diff
@@ -1402,7 +1402,10 @@ else if (actionType == ActionType.EXPLORE) {
 				subType = myKey.split(":")[1];
 			    }
 			}
-
+			if (subType.equalsIgnoreCase(":ALL")) {
+				meta = "ALL";
+				type = CMIMaterial.getGeneralMaterialName(type);
+			}

```

Comment:
You should put this check after `type == null` check. It throws NPE when the user given bad type.

Reply:
IDEs this can't reproduces, because of lot of code, or something else. If you try with using `dfdf-all` in jobConfig, you will see after a plugin reload that there is a NPE.

Reply:
Okay I simply moved that null check which includes the `continune;` before that block. So now we are really sure that there will ne be a NPE :)